unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/AbstractListTestCase.java

```diff
@@ -1132,4 +1134,20 @@ public void forEachWithIndex()
         Assert.assertEquals(FastList.newListWith(1, 2, 3, 4), elements);
         Assert.assertEquals(IntArrayList.newListWith(0, 1, 2, 3), indexes);
     }
+
+    @Test
+    public void replaceAll()
+    {
+        MutableList<Integer> integers = this.newWith(1, 2, 3, 4);
+        integers.replaceAll(i -> i * 2);
+        Assert.assertEquals(Lists.mutable.with(2, 4, 6, 8), integers);
+    }
+
+    @Test
+    public void sort()
+    {
+        MutableList<Integer> integers = this.newWith(1, 2, 3, 4);
+        integers.sort(Comparator.reverseOrder());
+        Assert.assertEquals(Lists.mutable.with(4, 3, 2, 1), integers);
+    }
```

Comment:
I think the thread-safety tests might be missing.

Reply:
Do the thread-safety tests need to be in Scala?

Reply:
We should eventually convert them all to Java.  I added them here to be consistent for now as we don't have an alternative yet.